cloud-storage/storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/AwsRequester.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.storage.storagetransfer.samples;
+
+import com.google.api.services.storagetransfer.Storagetransfer;
+import com.google.api.services.storagetransfer.model.AwsAccessKey;
+import com.google.api.services.storagetransfer.model.AwsS3Data;
+import com.google.api.services.storagetransfer.model.Date;
+import com.google.api.services.storagetransfer.model.GcsData;
+import com.google.api.services.storagetransfer.model.Schedule;
+import com.google.api.services.storagetransfer.model.TimeOfDay;
+import com.google.api.services.storagetransfer.model.TransferJob;
+import com.google.api.services.storagetransfer.model.TransferSpec;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+/**
+ * Creates a one-off transfer job from Amazon S3 to Google Cloud Storage.
+ *
+ */
+public final class AwsRequester {
+
+  private static final String JOB_DESC = "YOUR DESCRIPTION";
+  private static final String PROJECT_ID = "YOUR_PROJECT_ID";
+  private static final String AWS_SOURCE_NAME = "YOUR SOURCE BUCKET";
+  private static final String AWS_ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
+  private static final String AWS_SECRET_ACCESS_KEY = "YOUR_SECRET_ACCESS_KEY";
+  private static final String GCS_SINK_NAME = "YOUR_SINK_BUCKET";
+  private static final String START_DATE = "YYYY-MM-DD";
```

Comment:
You might wish to have a line break as folks aren't going to be setting this?  Or if they are, why?